packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.29+1
+
+* (ios) Pin dependency on GoogleMaps pod to `~>2.7`, to address https://github.com/flutter/flutter/issues/63447
```

Comment:
Looks like you are pinning on `3.9`, right?

Reply:
Haha yep